hicommonwealth/commonwealth
# #6513 - Update Editor Component UI - Toolbar Text Indenting

This PR adds buttons to the editor toolbar that allow the user to input left-aligned, center-aligned, right-aligned, and justified text. 

## Link to Issue
Closes: #6513 

## Description of Changes
- add buttons for text aligned to the left, center, right, and for justified text

## Test Plan
- go to the create thread page
- enter and select text. When you click on one of the text alignment buttons, your selected text will be wrapped in a styled `div`
- without selecting text, click on one of the text alignment buttons. A styled `div` will appear in the editor.
